android/app/webviewAssets.gradle

```diff
@@ -7,6 +10,45 @@
  * more information.
  */
 
+/**
+ * Windows support functions.
+ *
+ * On Windows, we have three different possible setups:
+ *   1. Windows Gradle + MSys Git's bash.
+ *   2. Windows Gradle + WSL's bash.
+ *   3. WSL Gradle + WSL's bash.
+ *
+ * Case 3, fortunately, should behave exactly like Linux.
+ *
+ * In cases 1 and 2, however, all paths Gradle provides are backslash-separated.
+ * This is problematic on its own, as the bash script we're invoking doesn't
+ * grok backslashes, so we normalize those on Windows. That's enough for case 1.
+ *
+ * Case 2 is worse, though. MSys's `bash` groks both "C:/" and "C:\"; more
+ * generally, it operates on the Windows filesystem with Windows path names. The
+ * WSL `bash` doesn't -- it's a standard Linux `bash` (or close enough), and it
+ * only sees the Windows disks as ordinary mounts under the lone volume '/'.
+ * Either "C:/" or "C:\" will confuse it terribly.
+ *
+ * And, of course, Gradle has no good way of distinguishing between cases 1 and
+ * 2. As far as it's concerned, `bash.exe` is `bash.exe`.
+ *
+ * Since we can't predict the environment of our callee, we can't use absolute
+ * paths. Fortunately, with only a little extra work in the bash script, we
+ * _can_ use relative paths, letting the script derelativize them appropriately.
+ */
+def normalizePath(String path) {
+    if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+        return path.replace('\\', '/')
+    } else {
+        return path
+    }
+}
+
+def relativizePath(String path, String src) {
+    return Paths.get(src).relativize(Paths.get(path)).toString();
```

Comment:
It's not obvious to me from the names which of these parameters is expected to be which.

Perhaps something like "path" and "base"? Or a line of javadoc could do it, too.